src/test/resources/com/puppycrawl/tools/checkstyle/checks/trailingcomment/InputTrailingCommentPastFalsePositive.java

```diff
@@ -0,0 +1,136 @@
+package com.puppycrawl.tools.checkstyle.checks.trailingcomment;
+
+/**
+ * Config:
+ * default
+ */
+public class InputTrailingCommentPastFalsePositive {
+    public static boolean mybool = true; // violation
+    /* FROM NOW ON EVERYTHING IN THIS INPUT FILE SHOULD TRIGGER NO TRAILING COMMENT VIOLATION!!! */
```

Comment:
Please remove it, this will be confusing/harder to maintain if we need to extend this input in future

Reply:
fixed